src/test/regress/expected/multi_metadata_sync.out

```diff
@@ -1656,8 +1655,7 @@ ORDER BY shardid, nodeport;
  shardid | nodename  | nodeport
 ---------------------------------------------------------------------
  1310073 | localhost |    57637
- 1310073 | localhost |    57638
```

Comment:
this seems to be ok, since it's mainly testing the integrity of the metadata and this is correctly reflecting that the reference table was not replicated yet